packages/image_picker/image_picker_ios/ios/Classes/FLTPHPickerSaveImageToPathOperation.m

```diff
@@ -184,4 +187,39 @@ - (void)processImage:(NSData *)pickerImageData API_AVAILABLE(ios(14)) {
   }
 }
 
+/**
+ * Processes the video.
+ */
+- (void)processVideo API_AVAILABLE(ios(14)) {
+  NSString *typeIdentifier = self.result.itemProvider.registeredTypeIdentifiers.firstObject;
+  [self.result.itemProvider
+      loadFileRepresentationForTypeIdentifier:typeIdentifier
+                            completionHandler:^(NSURL *_Nullable videoURL,
+                                                NSError *_Nullable error) {
+                              if (videoURL == nil) {
+                                return;
```

Comment:
In the image version, it returns a FlutterError when the expected data is nil. Should the same happen here?

Reply:
I'm just realizing I never went back and updated this block of code (it's pulled from an old pr a contractor wrote a while back).